docs/sampling.md

```diff
@@ -20,29 +20,26 @@ Tracer tracer = wfTracerBuilder.build();
 ```
 ## Supported Sampling Strategies
 
-The following table lists the supported sampling strategies. You create and configure a sampling strategy by configuring an `Sampler` implementation:
+The following table lists the supported sampling strategies. You create and configure a sampling strategy by configuring a `Sampler` implementation:
 
 | Sampler              | Description                            |
 | --------------------- | -------------------------------------- |
 | ConstantSampler       | Allows either all traces or no traces. Specify `true` to sample all traces, or `false` to sample no traces. |
 | DurationSampler       | Allows a span if its duration exceeds a specified threshold. Specify the duration threshold as a number of milliseconds. |
 | RateSampler           | Allows a specified probabilistic rate of traces to be reported. Specify the rate of allowed traces as a number between 0.0 and 1.0. |
-| CompositeSampler      | Delegates the sampling decision to multiple other samplers and allows a span if any delegate decides to allows it. Specify a list of samplers to delegate to. |
+| CompositeSampler      | Delegates the sampling decision to multiple other samplers and allows a span if any delegate decides to allows it. Specify a list of samplers to delegate. |
 
 
 **Note:** Regardless of the sampling strategy, the `WavefrontTracer`: 
 * Allows all error spans (`error=true` span tag).
 * Allows all spans that have a sampling priority greater than 0 (`sampling.priority` span tag).
 * Includes all spans in the [RED metrics](https://github.com/wavefrontHQ/wavefront-opentracing-sdk-java/blob/master/README.md#red-metrics) that are automatically collected and reported.
 
-
-
-
 ## Using Multiple Sampling Strategies
 
 You can configure a `WavefrontTracer` with multiple sampling strategies. To do so, you use a `CompositeSampler`, which delegates the sampling decision to multiple other samplers and decides to allow a span if any of the delegate samplers decide to allow it. 
 
-For instance, suppose you want to report approximately 10% of traces but you also don't want to lose any spans that are over 60 seconds long. The following code snippet shows how to configure a `WavefrontTracer` with a `RateSampler` and a `DurationSampler`:
+For instance, suppose you want to approximately report 10% of traces but you also don't want to lose any spans that are over 60 seconds long. The following code snippet shows how to configure a `WavefrontTracer` with a `RateSampler` and a `DurationSampler`:
```

Comment:
I British English, the adverb comes before the verb: "suppose you want to approximately report " 
But in American English, the adverb comes before the verb it's supposed to modify. So, "suppose you want to report approximately" . 
(same change above.)